src/components/talkCard/TalkCard.css

```diff
@@ -1,57 +1,96 @@
-  .container {
-    display: flex;
-    justify-content: center;
-  }
-  
-  /* Rectangle Card Layout */
-  .rectangle {
-    background: url(../../assets/images/talksCardBack.svg);
-    width: 380px;
-    height: 300px;
-    border-radius: 4px;
-    box-shadow: 0px 20px 50px #d9dbdf;
-    -webkit-transition: all 0.3s ease;
-    -o-transition: all 0.3s ease;
-    transition: all 0.3s ease;
-    margin-top: 50px;
-  }
-  .rectangle:hover {
-    box-shadow: 0 20px 40px rgba(0, 0, 0, 0.08);
-  }
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+/* Rectangle Card Layout */
+.rectangle {
+  background: url(../../assets/images/talksCardBack.svg);
+  width: 380px;
+  height: 300px;
+  border-radius: 4px;
+  box-shadow: 0px 20px 50px #d9dbdf;
+  -webkit-transition: all 0.3s ease;
+  -o-transition: all 0.3s ease;
+  transition: all 0.3s ease;
+  margin-top: 50px;
+}
+.rectangle:hover {
+  box-shadow: 0 20px 40px rgba(0, 0, 0, 0.08);
+}
+.dark-rectangle {
+  box-shadow: 4px 1px 20px 0px #ffffff;
```

Comment:
I don't think we need this, the shadow size isn't the same as light mode and if we do want to go with a smaller shadow, the shadow is off by a few pixels. 
`0px 0px 20px 0px` seems to fit right the box right with the following changes.

Before:

![image](https://user-images.githubusercontent.com/48270786/91814109-47d6ff80-ec51-11ea-82e0-33215b6a1145.png)


After:

![image](https://user-images.githubusercontent.com/48270786/91814074-3aba1080-ec51-11ea-8b74-f93a0931f783.png)